src/pydicom/pixels/encoders/native.py

```diff
@@ -62,6 +70,11 @@ def _encode_rle_frame(src: bytes, runner: EncodeRunner) -> bytes:
     if runner.get_option("byteorder", "<") == ">":
         raise ValueError("Unsupported option \"byteorder = '>'\"")
 
+    # In the case of single bit images, the data must first be bit-packed
+    # before being encoded with Deflate
```

Comment:
RLE

Reply:
resolved in [c1e9a0f](https://github.com/pydicom/pydicom/pull/2216/commits/c1e9a0f2c1b8225b608765a11be870fdd5d7fc99)